news/models.py

```diff
@@ -7,3 +7,81 @@ class Subscriber(models.Model):
 
     def __str__(self):
         return self.user.username
+
+
+class Reporter(models.Model):
+    user = models.ForeignKey('Subscriber', on_delete=models.CASCADE)
+    avatar = models.ImageField(blank=True, default=None, upload_to='reporters')
+    agency = models.ForeignKey('Agency', on_delete=models.CASCADE)
+
+    def __str__(self):
+        return self.user.user.username
+
+
+class Agency(models.Model):
+    name = models.CharField(max_length=250, unique=True)
+    description = models.TextField()
+    image = models.ImageField(upload_to='agencies')
+
+    def __str__(self):
+        return self.name
+
+
+class Category(models.Model):
+    title = models.CharField(max_length=50, unique=True)
+    description = models.CharField(max_length=250)
+    icon = models.ImageField(blank=True, default=None, upload_to='categories')
+    created_time = models.DateTimeField(auto_now_add=True)
+
+    def __str__(self):
+        return self.title
+
+
+class News(models.Model):
+    title = models.CharField(max_length=250)
+    agency = models.ForeignKey('Agency', on_delete=models.CASCADE)
+    # todo can be on_delete= set_null?
+    author = models.ForeignKey('Reporter', on_delete=models.CASCADE)
```

Comment:
I think It's better to not delete an author at all. Maybe just disable they're account or sth